tests/core/test_lightning_module.py

```diff
@@ -11,16 +11,68 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from unittest.mock import patch
+from unittest.mock import patch, Mock, PropertyMock
 
 import pytest
 from torch.optim import Adam, SGD
 
 from pytorch_lightning import Trainer
+from pytorch_lightning.loggers import TensorBoardLogger
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
 from tests.base import BoringModel
 
 
+def test_property_current_epoch():
```

Comment:
lest move it to separate file `tests/core/test_module_attribites.py`?

Reply:
hmm, not sure. Isn't it more logical that unit tests for pl/core/lightning_module go to tests/core/test_lightning?

Reply:
just try to keep the file light :]